### PR TITLE
updated the profit by genre

### PR DESCRIPTION
**Pull request by andywang219**
Removed plotly because you need an account

### CODE REVIEW

File: datasets.py [hunk 1]
```diff
@@ -182,3 +182,5 @@ def get_cast():
     cast_df = cast_df.drop('data', axis = 1)
 
     cast_df.to_csv("./clean_datasets/clean_cast.csv", index=False)
+    
+    return cast_df
```
Comment (khelif96) [inline datasets.py]: EOL

File: graphs.py [hunk 2]
```diff
@@ -105,106 +99,14 @@ def profit_by_genre():
...
-            image_filename='average_rating_of_each_genre',
-            output_type='file',
-            image_width=800,
-            image_height=600,
-            validate=False)
+    alt.Chart(genre_df).mark_bar().encode(
+        x='genre',
+        y='profit',
+        color='genre'
+    )
```
Comment (khelif96) [inline graphs.py]: EOL!!!!!!!!!!!!!!!

**Review by khelif96 — APPROVED** [review]
Fix the EOL stuff then your good to go.

File: datasets.py (hunk 1 above)
Comment (andywang219) [reply datasets.py]: Fixed double check

File: graphs.py (hunk 2 above)
Comment (andywang219) [reply graphs.py]: fixed, double check

**Comment by khelif96** [comment]
If this is done you can merge

**Pull request merged**